proposals/wac-ucr/index.bs

```diff
@@ -746,6 +746,17 @@ with conditions that stipulate:
     targeting the `measurements` [=collection=] where
     [=agents=] are included in the `interns` [=authorization group=].
 
+### Conditional access by action ### {#conditional-action}
+
+The University has a widely used blog for discussion with research teams
+across the world. Researchers who post comments often want to correct
+errors after publication. But this feature can lead to misunderstandings
+if edited after someone else has commented on the comment. They institute
+the following policy: their authors can edit comments for a minimum of 5
+minutes and only until these are themselves commented. After that point, the
+post will be in read-only access mode.
+
```

Comment:
@bblfish Is this another example of the conditional access by time use case, or is the conditional action that the author can edit the comment but only until someone responds to it?

Reply:
It's both. It seems like both are realistic.